tests/periph_flashpage/main.c

```diff
@@ -643,6 +688,7 @@ static const shell_command_t shell_commands[] = {
     { "edit", "Write bytes to the local page buffer", cmd_edit },
     { "test", "Write and verify test pattern", cmd_test },
     { "test_last_pagewise", "Write and verify test pattern on last page available", cmd_test_last },
+    { "test_res_pagewise", "Write and verify short write on reserved page", cmd_test_res},
```

Comment:
The name of this command is confusing: `res` can be confused with `result`. Better be explicit and call it `test_reserved_pagewise` or something the like.

Reply:
Fixed and I also added the command to `periph_flashpage/tests`